homeassistant/components/scene/homeassistant.py

```diff
@@ -25,41 +22,12 @@
 SceneConfig = namedtuple('SceneConfig', ['name', 'states'])
 
 
-def activate(hass, entity_id=None):
-    """ Activate a scene. """
-    data = {}
+# pylint: disable=unused-argument
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """ Sets up home assistant scene entries. """
+    scene_config = config.get("config")
```

Comment:
First, I know I'm not consistent and are going against what I said earlier, sorry about that. But I think that `config` is not a good description. Let's call it `states`.